### ISSUE TITLE
Make your journal more accessible.

### ISSUE THREAD

**Issue by devbisme** [algofoogle/journal#1]
Lots of good information in your journal entries! Lots.

But the presentation sucks! Just a list of markdown files that most people won't bother to click on.

Consider organizing them as a static site and hosting them on Github Pages inside this repository. You've already written the markdown files so most of the hard work is done.

If you do this, I think more people will see and benefit from all the work you've done.

**Comment by algofoogle** [comment]
Yeah good point, thanks for the feedback! The entries grew well beyond what I originally expected, so maybe now is the time.

What brought you here, btw? Random find, or maybe from a project?

**Comment by devbisme** [comment]
Apologies for the late reply. I don't come here often.

I run a Github repository searcher called [RepoRecon](https://devbisme.github.io/RepoRecon/) that scans for repositories about things I'm interested in. One of those things is KiCad and that matched with something in your `journal` repo. So I visited and saw a bunch of interesting stuff. Since I hate seeing useful information not get noticed (which was the main reason I made RepoRecon), I had to comment that you've got a hidden gold mine here.
